deployment/ansible/roles/driver.windshaft/templates/upstart-windshaft.conf.j2

```diff
@@ -16,7 +16,7 @@ pre-start script
   /usr/bin/docker rm windshaft || true
 
   {% if 'development' not in group_names -%}
-  /usr/bin/docker pull quay.io/azavea/windshaft:0.1.0
+  /usr/bin/docker pull {{ docker_repository }}/windshaft:0.1.0
```

Comment:
Hmm, this is an interesting wrinkle, I had forgotten about this. We don't actually build a windshaft image anywhere in the deployment process, we just use an existing Azavea image and use it to run `server.js`. So I think this should actually be hard-coded as `quay.io/azavea` (probably with a comment). Unless someone wanted to like, upgrade to a different version of Windshaft, they should be able to do everything they want by just modifying `server.js` so there'd be no need to use a custom docker image.